src/main/java/com/rybka/service/CSVExportService.java

```diff
@@ -0,0 +1,24 @@
+package com.rybka.service;
+
+import com.opencsv.CSVWriter;
+import com.rybka.model.CurrencyHistory;
+import lombok.extern.log4j.Log4j;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+@Log4j
+public class CSVExportService {
+
+    public void export(List<CurrencyHistory> histories, String file) {
```

Comment:
Did you consider to use interface at least for Csv/Json export services?

Reply:
fixed